tests/fxl-page-spread_center/OPS/nav.xhtml

```diff
@@ -0,0 +1,15 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:page-spread- center</title>
```

Comment:
Space after hyphen :) Applies to many other files in this PR, so I recommend searching this PR for "- " (hyphen space).